[INFO -- 2018-11-08 20:11:36]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-08 20:11:36]
Initialization strings:
[INFO -- 2018-11-08 20:11:36]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-08 20:11:36]
(2) set args /tmp/lreg/randInput /tmp/lreg/output /tmp/lreg/detected.log
[INFO -- 2018-11-08 20:11:36]
initSignal:0
[INFO -- 2018-11-08 20:11:36]
endSignal:10
[INFO -- 2018-11-08 20:11:36]
faultModel:1
[EXCEPTION -- 2018-11-08 20:11:42]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-11-08 20:11:42]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-11-08 20:11:43]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad88b7 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7adea06 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7adea41 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7adec74 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b06ed2 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x0000555555557fa0 in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd5f0, __n=18446744073709551608) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x0000555555557ced in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18446744073709551608) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x000055555555750e in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd5f0, __n=18446744073709551608) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x0000555555557273 in std::_Vector_base<double, std::allocator<double> >::_M_create_storage (this=0x7fffffffd5f0, __n=18446744073709551608) at /usr/include/c++/7/bits/stl_vector.h:187
#11 0x000055555555689b in std::_Vector_base<double, std::allocator<double> >::_Vector_base (this=0x7fffffffd5f0, __n=18446744073709551608, __a=...) at /usr/include/c++/7/bits/stl_vector.h:138
#12 0x0000555555556307 in std::vector<double, std::allocator<double> >::vector (this=0x7fffffffd5f0, __x=std::vector of length -8, capacity -8 = {...}) at /usr/include/c++/7/bits/stl_vector.h:328
#13 0x0000555555555713 in train (iterations=45000, alpha=2.2222199999999999e-05, N=4, stopsWhenStable=true) at lreg.cpp:126
#14 0x0000555555555a1c in execute (input=0x7fffffffe1f1 "/tmp/lreg/randInput") at lreg.cpp:157
#15 0x0000555555555c98 in main (argc=4, argv=0x7fffffffde48) at lreg.cpp:194

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping value of field: __val
Fliping array at pos: 0
Memory content before bitflip:0x000x000x000x000x000x000x000x00
Fault Model: Double bit-flip
base address to flip value: 0x7fffffffd160
address max offset: 8
Memory content after  bitflip:0x000x000x000x000x000x000x000x0a
frame name: __GI_raise
symbol name: set
symbol filename: ../sysdeps/unix/sysv/linux/raise.c
symbol line: 39
value: {__val = {720575940379279360, 0, 4613937818241073152, 0, 4602678819172646912, 0, 13717591962318260970, 0, 4347869596288417792, 0, 4418972514053300350, 0, 0, 0, 0, 0}}
value address: 0x7fffffffd160
Type: The type is a structure.
Type sizeof: 128
Field name: __val
Field Type: The type is an array.
Field Type sizeof: 128

[INFO -- 2018-11-08 20:11:43]
Fault Injection Successful after 7.107951641082764s
[INFO -- 2018-11-08 20:11:43]
event type: exit
[EXCEPTION -- 2018-11-08 20:11:43]
exit code: no exit code available
